dist/query_ctrl.ts

```diff
@@ -47,6 +49,27 @@ export class MetaQueriesQueryCtrl extends QueryCtrl {
 
     this.queryTypes = _.keys(this.queryTypeValidators);
 
+    if (!this.target.orderType) {
+			this.target.orderType = 'Top';
+    }
+    
+		this.orderTypes = ['Top', 'Bottom'];
+ 
+		if (!this.target.orderSize) {
```

Comment:
This code makes it impossible to save an arithmetic metric with orderSize 'no limit' (id == 0).

Reply:
This bug also fixed by !this.target.orderSize && this.target.orderSize !== 0